doc/manual/citing_aspect.bib

```diff
@@ -306,3 +321,67 @@ @techreport{Bangerth2017a
 url = {https://doi.org/10.6084/m9.figshare.4865333},
 year = {2017}
 }
+@article{Bredow2018a,
```

Comment:
I think I accidentally duplicated this entry in the Mendeley group and now it appears twice in the bib file. I removed the duplicate copy from Mendeley, could you do the same here? Sorry for that.